RealmSwift/Sync.swift

```diff
@@ -1274,6 +1274,41 @@ extension Realm {
     public func getPrivileges(forClassNamed className: String) -> ClassPrivileges {
         return ClassPrivileges(rawValue: RLMGetComputedPermissions(rlmRealm, className))
     }
+
+    /**
+    Returns the class-wide permissions for the given class.
+
+    This will return nil for non-partial-sync Realms.
```

Comment:
This doesn't seem to be true?

Reply:
Oops, I apparently only removed that from one of the two functions when I changed my mind on this point.